docs/guides/searching-with-readthedocs.rst

```diff
@@ -1,48 +1,20 @@
 Searching with Read the Docs
 ============================
 
-Read the Docs uses Elasticsearch to provide a better search experience.
+Read the Docs uses :doc:`/server-side-search` to provide a better search experience.
```

Comment:
A holdover from the original text, but "better" doesn't refer to anything here.

Reply:
I think it's comparing it to the default search builtin to the tooling.

Reply:
Yeah, it's just not clear to the reader that we're referring to the built in sphinx search